languages/java/oso/pom.xml

```diff
@@ -71,6 +71,15 @@
 
   <build>
   <plugins>
+    <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Why's this here? Looks related to testing?

Reply:
Yes, it's to make the Java tests spit out stack traces instead of silently swallowing them so you can't see what actually failed.